builtin_interfaces/Time stamp
string node_name
uint32 pub_idx